Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -3251,7 +3256,7 @@ void drawFilSetMenu() {
   void drawLedControlMenu() {
     checkkey = ID_Menu;
     if (SET_MENU(ledControlMenu, MSG_LED_CONTROL, 10)) {
-      BACK_ITEM(drawControlMenu);
+      BACK_ITEM((currentMenu == tuneMenu) ? drawTuneMenu : drawControlMenu);
```

Comment:
Is it working?
This will go back to drawControlMenu because for the BACK_ITEM the currentMenu will be drawLedControlMenu.